SolutionInfo.cs

```diff
@@ -34,3 +34,6 @@
 [assembly: AssemblyProduct("http://www.dnnsoftware.com")]
 [assembly: AssemblyCopyright("DotNetNuke is copyright 2002-2018 by DNN Corporation. All Rights Reserved.")]
 [assembly: AssemblyTrademark("DNN")]
+[assembly: AssemblyVersion("9.4.2")]
+[assembly: AssemblyFileVersion("9.4.2.5835")]
+[assembly: AssemblyInformationalVersion("9.4.2-unstable.5835+Branch.reorg3.Sha.a7727fcf1e2a6d08b3b7fb1b110401296886fb96")]
```

Comment:
What do you envision as the process for changing the version that's in source control?  Is that a manual step of running a Cake task after each release and committing the result?

Reply:
This is the "victim" of our change. There doesn't seem to be a perfect solution for now. The thing is: if someone clones the repo it should work when you build a debug version so you can debug. Before this PR you'd need to run GitVersion. So now you don't need to do this any longer.

On the downside: we'll need to update this file for every release. I guess this could be an automatic process.

Reply:
Didn't we discuss this so that each project would individually set it's version using the root GitVersion info on build (debug or release) and revert it's own manifest and the end of build ? This would make it work for both release and debug, without duplication and without commiting the version number. If we do commit version numbers, then we will keep having merge conflicts every time we merge the release branch into the development branch... At least until we reorganize the branchs which cannot happen until Dnn10.

Reply:
No, that is not what we settled on. We settled on "floating" versions from now on to facilitate local development.

We've wrestled with this for a while now. If you rely on GitVersion to build in debug locally it is a non starter. The solutioninfo will only get updated when doing a packaging build because that calls GitVersion. For local bebug building the solutioninfo will not change.

Reply:
Oh, yeah...

Reply:
NP